From 4923c563ccdd9d5399e901be03e528d80d896d1a Mon Sep 17 00:00:00 2001
From: Mathias Kresin <dev@kresin.me>
Date: Sun, 12 Jun 2016 11:14:09 +0200
Subject: [PATCH] mtd: nandsim: move false positive warnings to the info log
 level

The following warnings appear directly after loading the nandsim
driver:

[ 1388.386836] [nandsim] warning: read_byte: unexpected data output cycle, state is STATE_READY return 0x0
[ 1388.396900] [nandsim] warning: read_byte: unexpected data output cycle, state is STATE_READY return 0x0
[ 1388.407224] [nandsim] warning: read_byte: unexpected data output cycle, state is STATE_READY return 0x0
[ 1388.418124] [nandsim] warning: read_byte: unexpected data output cycle, state is STATE_READY return 0x0
[ 1388.429526] [nandsim] warning: write_byte: command (0x90) wasn't expected, expected state is STATE_READY, ignore previous states
[ 1388.962452] [nandsim] warning: write_byte: command (0x0) wasn't expected, expected state is STATE_READY, ignore previous states

Signed-off-by: Mathias Kresin <dev@kresin.me>
---
 drivers/mtd/nand/nandsim.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/mtd/nand/nandsim.c
+++ b/drivers/mtd/nand/nandsim.c
@@ -1921,7 +1921,7 @@ static u_char ns_nand_read_byte(struct m
 		return outb;
 	}
 	if (!(ns->state & STATE_DATAOUT_MASK)) {
-		NS_WARN("read_byte: unexpected data output cycle, state is %s "
+		NS_INFO("read_byte: unexpected data output cycle, state is %s "
 			"return %#x\n", get_state_name(ns->state), (uint)outb);
 		return outb;
 	}
@@ -2017,7 +2017,7 @@ static void ns_nand_write_byte(struct mt
 				 * was expected but command was input. In this case ignore
 				 * previous command(s)/state(s) and accept the last one.
 				 */
-				NS_WARN("write_byte: command (%#x) wasn't expected, expected state is %s, "
+				NS_INFO("write_byte: command (%#x) wasn't expected, expected state is %s, "
 					"ignore previous states\n", (uint)byte, get_state_name(ns->nxstate));
 			}
 			switch_to_ready_state(ns, NS_STATUS_FAILED(ns));
